openslides_backend/action/actions/user/create_update_permissions_mixin.py

```diff
@@ -196,56 +197,64 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
         if permstore.user_oml == OrganizationManagementLevel.SUPERADMIN:
             return
 
+        # store scope, id and OML-permission for requested user
+        if uid := instance.get("id"):
+            self.apply_instance(instance)
+
+        (
+            self.req_user_scope,
```

Comment:
The "request user" in our wording is always the user which issues the request, who is saved into `self.user_id`. Using the same wording now for the user indicated in the request is highly confusing. Maybe use something like `instance_user`?

Reply:
done